test/run_zones.jl

```diff
@@ -164,4 +166,12 @@ for i in 1:10
     end
 end
 
+ran = Ref(false)
+@tracepoint "disabled" enabled=false begin
+    set_zone_name!("disabled")
+    set_zone_text!("disabled")
+    ran[] = true
+end
+@test ran[]
```

Comment:
I added this test because I was uncertain of the semantics, but it goes counter to the logic I see implemented in tracepoint.jl 

In particular I was hopeing for:
1. The body of the tracepoint is executed in any circumstance
2. Using `set_zone_text!` is legal when the zone is disabled.

Such that I could use it to annotate a zone with more metadata.

Reply:
I was more curious what the behavior ought to be. Because I was surprised that this test passed.
If I read the code right I would expect `set_zone_name!` to throw.